usage/src/com/cloud/usage/UsageSanityChecker.java

```diff
@@ -43,9 +43,9 @@
     protected static final int DEFAULT_AGGREGATION_RANGE = 1440;
     protected StringBuilder errors;
     protected List<CheckCase> checkCases;
-    protected String lastCheckFile = "/usr/local/libexec/sanity-check-last-id";
+    protected String lastCheckFile = "/var/cache/cloudstack/usage/sanity-check-last-id";
```

Comment:
Just a thought, why we store this in a file (on one of the mgmt/usage server hosts) and not in database?